File: 283-move-zeroes/move-zeroes.cs
public class Solution {
    public void MoveZeroes(int[] nums) {

        List<int> dummy = new List<int>();
			int zerocounter = 0;
			foreach (int iter in nums)
			{
				if (iter == 0)
				{
					zerocounter++;
				}
				else
				{
					dummy.Add(iter);
				}
			}

			for (int i = 0; i < zerocounter; i++)
			{
				dummy.Add(0);
			}


			for (int i = 0; i < nums.Length; i++)
			{
				nums[i] = dummy[i];
				
			}

		

        
    }
}

File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        val = nums.count(0)
        arr = []
        for i in (nums):
            if(i != 0 ):
                arr.append(i)

        for i in range(len(arr)):
            nums[i] = arr[i]

        for i in range(len(arr) , len(nums)):
            nums[i] = 0
        

        